### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by SoftwareEngineer66**
No description provided

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,12 @@
...
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Welcome to Potionfy</title>
+</head>
+<body>
+  <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (LeadEngineer66) [inline index.html:10]: Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

Comment (SoftwareEngineer66) [reply index.html:10]: Sure can do!

**Review by LeadEngineer66 — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by LeadEngineer66 — APPROVED** [review]
Well done Harry!

**Pull request merged**